eclipse-scout-core/src/util/files.js

```diff
@@ -48,7 +48,28 @@ export function fileListToArray(fileList) {
   return files;
 }
 
+export function getErrorMessageMaximumUploadSizeExceeded(session, maximumUploadSize) {
+  if (maximumUploadSize < 1024) {
+    return session.text('ui.FileSizeLimit', maximumUploadSize, session.text('ui.Bytes'));
+  }
+  let maximumUploadSizeKB = maximumUploadSize / 1024;
+  if (maximumUploadSizeKB < 1024) {
+    return session.text('ui.FileSizeLimit', roundToTwoDecimals(maximumUploadSizeKB), 'KB');
+  }
+  let maximumUploadSizeMB = maximumUploadSizeKB / 1024;
+  if (maximumUploadSizeMB < 1024) {
+    return session.text('ui.FileSizeLimit', roundToTwoDecimals(maximumUploadSizeMB), 'MB');
+  }
+  return session.text('ui.FileSizeLimit', roundToTwoDecimals(maximumUploadSizeMB / 1024), 'GB');
+}
+
+function roundToTwoDecimals(number) {
```

Comment:
I'd suggest you use Scout's numbers-Utility #round method. As the JS-Doc there says: <<Numbers should not be rounded with the built-in Number.toFixed() method, since it behaves differently on different browsers>>